crates/holochain/src/core/workflow/incoming_dht_ops_workflow.rs

```diff
@@ -142,10 +137,8 @@ pub async fn incoming_dht_ops_workflow(
         }
     }
 
-    if !request_validation_receipt {
```

Comment:
I'm confused by this check. At first I thought if validation receipts are always requested now, this filtering should be deleted. But it makes sense to filter out ops that are already in the database. But why was this conditional before?